src/Type/EqualType.php

```diff
@@ -20,15 +20,15 @@
 use Symfony\Contracts\Translation\TranslatorInterface;
 
 @trigger_error(
-    'The '.__NAMESPACE__.'\EqualType class is deprecated since version 1.2 and will be removed in 2.0.'
+    'The '.__NAMESPACE__.'\EqualType class is deprecated since version 0.x and will be removed in 2.0.'
```

Comment:
This is a bit strange, it was deprecated on 1.x? should we undeprecate on 0.x? Or it is deprecated already on Core?

Reply:
It is little strange becouse we will release 0.x after 1.x. Also contributing proces from CoreBundle to extensions are strange. Branch 0.x should be trait as CoreBundle part which allow upgrade to this bundle to extensions (^1.4). We should mark deprecations for that.